graph/telemetry/istio/appender/ambient.go

```diff
@@ -33,12 +33,10 @@ func (a AmbientAppender) AppendGraph(trafficMap graph.TrafficMap, globalInfo *gr
 
 	log.Trace("Running ambient appender")
 
-	if !a.Waypoints {
-		a.removeWaypointEntries(trafficMap, globalInfo, namespaceInfo)
-	}
+	a.isWaypoint(trafficMap, globalInfo, !a.Waypoints)
 }
 
-func (a AmbientAppender) removeWaypointEntries(trafficMap graph.TrafficMap, globalInfo *graph.AppenderGlobalInfo, namespaceInfo *graph.AppenderNamespaceInfo) {
+func (a AmbientAppender) isWaypoint(trafficMap graph.TrafficMap, globalInfo *graph.AppenderGlobalInfo, remove bool) {
```

Comment:
Very minor, but `isWaypoint` seems like a function that would test something and return a bool.  Maybe rename to something like `processWaypoints` or `handleWaypoints` ?

Reply:
Changed to handleWaypoints